plone/dexterity/schema.py

```diff
@@ -326,7 +329,12 @@ def portalTypeToSchemaName(portal_type, schema=u"", prefix=None, suffix=None):
     """Return a canonical interface name for a generated schema interface.
     """
     if prefix is None:
-        prefix = '/'.join(getUtility(ISiteRoot).getPhysicalPath())[1:]
+        if portal_type == 'Plone Site':
+            fti = getUtility(IDexterityFTI, name=portal_type)
```

Comment:
If we use `queryUtility` here, and catch the case where it is None, then this PR could probably be merged already before all the other dexterity site root PRs. It is a guess.

Reply:
Done.